lib/grape-swagger/endpoint.rb

```diff
@@ -287,17 +287,13 @@ def expose_params_from_model(model)
       model_name
     end
 
-    def model_name(name)
-      if name.respond_to?(:entity_name)
-        name.entity_name
-      elsif name.to_s.end_with?('Entity', 'Entities')
-        length = 0
-        name.to_s.split('::')[0..-2].reverse.take_while do |x|
-          length += x.length
-          length < 42
-        end.reverse.join
+    def model_name(entity)
+      if entity.respond_to?(:entity_name)
+        entity.entity_name
+      elsif entity.to_s.end_with?('::Entity', '::Entities')
+        entity.to_s.split('::')[-2]
```

Comment:
btw @LeFnord why do we check for `end_with?` here, but removing all 'Entity' string parts here https://github.com/ruby-grape/grape-swagger/blob/master/lib/grape-swagger/doc_methods/data_type.rb#L49 
Which algo is correct?